docs/markdown/Python/python-goals/python-test-goal.md

```diff
@@ -354,6 +354,13 @@ If you use multiple files with `test --debug`, they will run sequentially rather
 >
 > Run your test with `./pants test --debug` as usual.
 
+Failures to collect tests
+-------------------------
+
+`pytest` follows [certain conventions for test discovery](https://docs.pytest.org/en/7.1.x/explanation/goodpractices.html#conventions-for-python-test-discovery), so if no (or only some) tests are run, it may be worth reviewing the documentation. Pants can help you find test modules that would not be collected by `pytest`. For instance, `./pants tailor --check ::` command would suggest creating targets for files that are not covered by glob expressions in your `BUILD` files (e.g. if a test module has a typo and is named `tes_connection.py`). You can also run `./pants filedeps <tests-dir>::` command to list all files known to Pants and compare the output with the list of files that exist on disk.
```

Comment:
How about `./pants --filter-target-type=python_test filedeps <test-dir>::`?

Reply:
Even better, thank you!